Block/Stub.php

```diff
@@ -69,4 +96,25 @@ public function getAbstractObject()
     {
         return null;
     }
+
+    /**
+     * Returns if autoloading recommendations is disabled or not.
+     *
+     * @return boolean
+     */
+    public function isRecoAutoloadDisabled()
+    {
+        $store = $this->getNostoHelperScope()->getStore(true);
+        // If price variations are used and the variation something else than
+        // the default one we disable the autoload. For default variation
+        // the sections are not loaded and loadRecommendations() is not called
+        if ($this->nostoHelperData->isPricingVariationEnabled($store)
+            && !$this->nostoHelperVariation->isDefaultVariationCode(
+                $this->nostoHelperCustomer->getGroupCode()
```

Comment:
Missing try catch or `@throws LocalizedException` and `@throws NoSuchEntityException` at the docblock

Reply:
We'll let it blow up. We want it to be obvious that the price variations cannot be used.